src/Bridges/rsocbridge.jl

```diff
@@ -28,8 +28,8 @@ function RSOCBridge{T}(model, f::MOI.VectorAffineFunction{T}, s::MOI.RotatedSeco
     u = MOIU.eachscalar(f)[2]
     x = MOIU.eachscalar(f)[3:d]
     s2 = √2
-    y = MOI.ScalarAffineFunction([t.variables; u.variables], [t.coefficients/s2; -u.coefficients/s2], t.constant/s2 - u.constant/s2)
-    z = MOI.ScalarAffineFunction([t.variables; u.variables], [t.coefficients/s2;  u.coefficients/s2], t.constant/s2 + u.constant/s2)
+    y = MOI.ScalarAffineFunction([mapcoefficient.(c -> c/s2, t.terms); mapcoefficient.(c -> -c/s2, u.terms)], t.constant/s2 - u.constant/s2)
```

Comment:
`mapcoefficient.(c -> c/s2, [t.terms; u.terms])`?
Same below

Reply:
The maps is different, the first map is `c/s2` and the second map is `-c/s2`.

Reply:
Oh, ok.